lib/senders/templates/verify_primary.txt

```diff
@@ -0,0 +1,12 @@
+{{t "Verify primary email"}}
+{{t "A request to perform an account change has been made from the following device:"}}
+{{ device }}
+{{#if location}}{{ location }}{{/if}}
+{{#if ip}}{{t "IP address: %(ip)s" }}{{/if}}
+{{t "Verify email: "}} {{{link}}}
+
+{{{t "This is an automated email;if you did not authorize this action, then <a %(passwordChangeLinkAttributes)s>please change your password</a>."}}}
+{{{t "For more information, please visit <a %(supportLinkAttributes)s>Mozilla Support</a>."}}}s
```

Comment:
Why are there `<a>` elements in the text template? These should be deleted right?